### PR TITLE
Finalpopup

### PR DESCRIPTION
**Pull request by hswiergiel**
adding the router and making a redirection to the score page when the game is over

### CODE REVIEW

**Review by guenk — APPROVED** [review]
evil dead II

File: frontend/src/assets/storm_gust/Readme!.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+By installing or using this font, you are agree to the Product Usage Agreement:
```
Comment (LoicBrassart) [inline frontend/src/assets/storm_gust/Readme!.txt:1]: Did you read this file ?

File: frontend/src/components/Timer/Timer.jsx [hunk 2]
```diff
@@ -1,21 +1,38 @@
...
 import "react-toastify/dist/ReactToastify.css";
 import PropTypes from "prop-types";
 
-export default function Countdown(props) {
-  const { startingMinutes = 1, startingSeconds = 30 } = props;
-  const [mins, setMinutes] = useState(startingMinutes);
-  const [secs, setSeconds] = useState(startingSeconds);
+export default function Countdown({ counter }) {
+  const [mins, setMinutes] = useState(0);
+  const [secs, setSeconds] = useState(2);
```
Comment (LoicBrassart) [inline frontend/src/components/Timer/Timer.jsx:9]: Not important, but could have been done with one state only

File: frontend/src/pages/Home/Home.jsx [hunk 3]
```diff
@@ -1,12 +1,34 @@
...
   return (
     <HomeStyle>
-      <Header />
       <img src={logo} alt="Logo Tech Legend" />
+      <h2>
+        Régles <span>du Jeu</span>
+      </h2>
+      <p>
+        Bienvenue dans le blind test proposé par les <span>Tech Legend</span> .
+        Mais qu'est ce qu'un blind test me dirais vous?
```
Comment (LoicBrassart) [inline frontend/src/pages/Home/Home.jsx:14]: "Me direz vous" :p

**Review by LoicBrassart — APPROVED** [review]
Beware, there are quite a lot of typos, plz fix this in a next PR

File: frontend/src/assets/storm_gust/Readme!.txt (hunk 1 above)
Comment (hswiergiel) [reply frontend/src/assets/storm_gust/Readme!.txt:1]: well, it's not commercial use, is it? :)
anyway, some changes have been made on the graphic render so i guess we'll suppress this font in the next pushes

File: frontend/src/components/Timer/Timer.jsx (hunk 2 above)
Comment (hswiergiel) [reply frontend/src/components/Timer/Timer.jsx:9]: alright!

File: frontend/src/pages/Home/Home.jsx (hunk 3 above)
Comment (hswiergiel) [reply frontend/src/pages/Home/Home.jsx:14]: on va modifier ça!

**Pull request merged**